src/libraries/System.Data.Common/src/System/Data/LinqDataView.cs

```diff
@@ -230,11 +233,13 @@ internal override void SetIndex(string newSort, DataViewRowState newRowStates, I
         #endregion
 
         #region IBindingList
-// TODO: Enable after System.ComponentModel.TypeConverter is annotated
+        // TODO: Enable after System.ComponentModel.TypeConverter is annotated
 #nullable disable
         /// <summary>
         /// Clears both expression-based and DataView's string-based sorting.
         /// </summary>
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
```

Comment:
What is unsafe about the two methods below? Can these suppressions (and thus the RUC on the ctor) be removed on this class?